Scripts/RemediationScripts/AutoRemediation.ps1

```diff
@@ -156,6 +156,8 @@ Enter the choice (1|2)";
                 ($control.ControlId -eq "Azure_APIManagement_DP_Use_HTTPS_URL_Scheme") -or
                 ($control.ControlId -eq "Azure_CloudService_SI_Disable_RemoteDesktop_Access") -or
                 ($control.ControlId -eq "Azure_ServiceFabric_DP_Set_Property_ClusterProtectionLevel") -or
+                ($control.ControlId -eq "Azure_SQLDatabase_DP_Enable_TDE") -or
```

Comment:
This Azure_SQLDatabase_DP_Enable_TDE is duplicate.

Reply:
fixed